dev/TOC.md

```diff
@@ -61,6 +61,7 @@
   + Configure
     - [Time Zone](how-to/configure/time-zone.md)
     - [Memory Control](how-to/configure/memory-control.md)
+    - [Cluster Initialization](how-to/configure/initialize-cluster.md)
```

Comment:
```suggestion
    - [Initialize a TiDB Cluster](how-to/configure/initialize-cluster.md)
```

Reply:
As the parent topic is Configure, I think we should use a noun phrase here.

Reply:
Discussed with @lilin90. We think we can use "Cluster Initialization" in TOC.md, and "Initialize a TiDB Cluster" as the document title, because it is in the how-to category.